### PR TITLE
PD-598 px backup changes

### PR DESCRIPTION
**Pull request by nathan-px**

### CODE REVIEW

File: content/install.md [hunk 1]
```diff
@@ -48,17 +53,21 @@ Create a Kubnernetes secret, save the name and namespace in which it's located f
 
 ## Install
 
-1. To install PX-Backup, generate the install script through the **PX-Backup using PX-Central** [spec generator](https://central.portworx.com/specGen/wizard). If you saved your cloud credentials as a Kubernetes secret ahead of time, enter the name and namespace of your secret.
+1. To install PX-Backup, generate the install spec through the **PX-Backup** [spec generator](https://central.portworx.com/specGen/wizard).
```
Comment (nathan-px) [inline content/install.md:56]: @bhavana-ar Do we need to add to this instruction, telling users how to add the PX storageclass name if they want to use PX?

File: content/install.md [hunk 2]
```diff
@@ -11,9 +11,15 @@ series: backup
 ## Prerequisites
 
 * Stork 2.4.0 or newer
-* {{< pxEnterprise >}} 2.5.0 or newer
-* PX-Central. Refer to the [Install PX-Central on-premises](https://docs.portworx.com/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/pxcentral-onprem/install-pxcentral/) page for details.
 * If you're using an external OIDC provider, you must use certificates signed by a trusted certificate authority.
+* Helm
+
+<!--  I think we can remove this now:
```
Comment (nathan-px) [inline content/install.md:17]: can we remove this requirement for new installs?

File: content/install.md [hunk 3]
```diff
@@ -24,6 +30,8 @@ series: backup
 
 ## Save your cloud credentials in a Kubernetes secret (Optional)
 
+<!-- is this still relevant? -->
```
Comment (nathan-px) [inline content/install.md:33]: is the cloud credentials as a Kubernetes secret section still relevant?

File: content/use-px-backup/upgrade.md [hunk 4]
```diff
@@ -9,4 +9,91 @@ scrollspy-container: false
...
+<!-- What is the name here? is it helm specific? I assume it's not namespace -->
+
+    ```text
+    app.kubernetes.io/name: <your-PX-Backup-name>
+    app.kubernetes.io/version: <your-current-PX-Backup-version>
+    ```
+
+3. Change the following annotations, specifying:
+
+    <!-- * need a description for each of these -->
```
Comment (nathan-px) [inline content/use-px-backup/upgrade.md:32]: we need a clear description of what users need to put for each field.

File: content/use-px-backup/upgrade.md [hunk 5]
```diff
@@ -9,4 +9,91 @@ scrollspy-container: false
...
+    ```text
+    curl -o create_volume_snapshot_from_pvc.yaml https://raw.githubusercontent.com/portworx/helm/chart_upgrade_specs/pxcentral-operator-to-chart-upgrade/create_volume_snapshot_from_pvc.yaml
+    ```
+
+2. Edit the spec, changing all occurrences of the `namespace: portworx` field to the namespace in which your current on-prem PX-Central deployment exists.
+
+## Upgrade you operator-based on-prem PX-Central deployment to the Helm chart method
+
+1. Create a volume snapshot from existing PX-Central on-prem PVCs in the same namespace:
+<!-- I don't understand what this sentence is trying to say -->
```
Comment (nathan-px) [inline content/use-px-backup/upgrade.md:53]: I can't quite understand what this and the step directly below it are trying to communicate.

File: content/use-px-backup/upgrade.md [hunk 6]
```diff
@@ -9,4 +9,91 @@ scrollspy-container: false
...
+    $ kubectl apply -f create_volume_snapshot_from_pvc.yaml
+    ```
+
+2. Verify that the volume snapshot for all PVCs is available in the same namespace:
+<!-- I don't understand what this sentence is trying to say -->
+
+    ```console
+    $ storkctl get snapshot --namespace portworx
+    ```
+    <!-- What's the expected output here? I assume users should see the upgrade snapshot -->
```
Comment (nathan-px) [inline content/use-px-backup/upgrade.md:65]: we should show an expected output for the verify step

File: content/install.md (hunk 2 above)
Comment (nathan-px) [reply content/install.md:17]: If users are running portworx with px-backuop, they need to use px enterprise 2.5.0 or newer.
px-central is ok to be removed.

File: content/install.md (hunk 3 above)
Comment (nathan-px) [reply content/install.md:33]: this no longer applies, remove it

File: content/install.md (hunk 1 above)
Comment (nathan-px) [reply content/install.md:59]: if users want to use Portworx with px-backup, they need to first create a storage class on the kubernetes cluster, select the`use storageclass` check box on the spec gen, and provide the storage class name to the spec gen. 

the storageclass they create on the kubernetes cluster beforehand looks something like this: 

```
kind: StorageClass
apiVersion: storage.k8s.io/v1
metadata:
    name: portworx-sc
provisioner: kubernetes.io/portworx-volume
parameters:
   repl: "3"
```

File: content/use-px-backup/upgrade.md [hunk 7]
```diff
@@ -9,4 +9,91 @@ scrollspy-container: false
...
+    app.kubernetes.io/version: <your-current-PX-Backup-version>
+    ```
+
+3. Change the following annotations, specifying:
+
+    <!-- * need a description for each of these -->
+
+    ```text
+    meta.helm.sh/release-name: px-backup
+    meta.helm.sh/release-namespace: portworx
```
Comment (nathan-px) [inline content/use-px-backup/upgrade.md:36]: the only thing users need to change in this step and the one above it, is this release-namespace. They need to add whatever namespace they installed px-central/px-backup onto originally.

File: content/use-px-backup/upgrade.md (hunk 4 above)
Comment (nathan-px) [reply content/use-px-backup/upgrade.md:32]: we can remove this step

File: content/use-px-backup/upgrade.md (hunk 5 above)
Comment (nathan-px) [reply content/use-px-backup/upgrade.md:53]: this creates a volume snapshot from the existing PX-Central on-prem PVCs in the namespace provided above:

File: content/use-px-backup/upgrade.md [hunk 8]
```diff
@@ -9,4 +9,91 @@ scrollspy-container: false
...
+## Upgrade you operator-based on-prem PX-Central deployment to the Helm chart method
+
+1. Create a volume snapshot from existing PX-Central on-prem PVCs in the same namespace:
+<!-- I don't understand what this sentence is trying to say -->
+
+    ```console
+    $ kubectl apply -f create_volume_snapshot_from_pvc.yaml
+    ```
+
+2. Verify that the volume snapshot for all PVCs is available in the same namespace:
```
Comment (nathan-px) [inline content/use-px-backup/upgrade.md:59]: this verifies that the snapshots you've taken in the step above are intact and ready to be restored.

File: content/use-px-backup/upgrade.md (hunk 6 above)
Comment (nathan-px) [reply content/use-px-backup/upgrade.md:65]: bhanava to send output, users output will similar to this. The only difference would be the namespace, if they're using a non-default namespace.

File: content/use-px-backup/upgrade.md [hunk 9]
```diff
@@ -9,4 +9,91 @@ scrollspy-container: false
...
+<!-- I don't understand what this sentence is trying to say -->
+
+    ```console
+    $ storkctl get snapshot --namespace portworx
```
Comment (nathan-px) [inline content/use-px-backup/upgrade.md:63]: tell them to use whatever namespace they've chosen originally, if they used a non-standard namespace in their previous installation.

Comment (nathan-px) [inline content/use-px-backup/upgrade.md:63]: remove `$` bash symbol

Comment (nathan-px) [reply content/use-px-backup/upgrade.md:63]: place the namespace example here into carrot brackets so users can't just paste

File: content/use-px-backup/upgrade.md [hunk 10]
```diff
@@ -9,4 +9,91 @@ scrollspy-container: false
...
+    $ bash px-central-cleanup.sh
+    ```
+
+4. Create new PVCs from the volume snapshots you created in step 1 in the same namespace where PX-Central on-prem was originally deployed:
+
+    ```console
+    $ kubectl apply -f create_new_pvc_from_snapshot.yaml
+    ```
+
+5. Install PX-Backup using the helm chart:
```
Comment (nathan-px) [inline content/use-px-backup/upgrade.md:89]: before this step, add another one telling users to retrieve the org name from the previous release.

Comment (nathan-px) [reply content/use-px-backup/upgrade.md:89]: @bhavana-ar to provide this step.

File: content/use-px-backup/upgrade.md [hunk 11]
```diff
@@ -9,4 +9,91 @@ scrollspy-container: false
...
+
+    ```console
+    $ helm repo add portworx http://charts.portworx.io/
+    $ helm repo update
+    $ helm install px-backup portworx/px-backup --namespace portworx --create-namespace --set persistentStorage.enabled=true,persistentStorage.storageClassName=stork-snapshot-sc,operatorToChartUpgrade=true,pxbackup.orgName=test,pxcentralDBPassword=singapore
```
Comment (nathan-px) [inline content/use-px-backup/upgrade.md:94]: carrot brace out the org name `orgName=test` and the `--namespace`

File: content/use-px-backup/upgrade.md (hunk 6 above)
Comment (bhavana-ar) [reply content/use-px-backup/upgrade.md:65]: We need to provide the instruction to install storkctl here: https://docs.portworx.com/portworx-install-with-kubernetes/disaster-recovery/px-metro/1-install-px/#install-storkctl

The output of storkctl would look like this:

File: content/use-px-backup/upgrade.md (hunk 10 above)
Comment (bhavana-ar) [reply content/use-px-backup/upgrade.md:89]: The cmd to fetch the org name from 1.0.3 cluster is `kubectl get cm pxc-central-ui-configmap -o yaml -n <px-central-namespace> | grep PX_BACKUP_ORGID`

File: content/use-px-backup/upgrade.md (hunk 11 above)
Comment (bhavana-ar) [reply content/use-px-backup/upgrade.md:94]: helm install px-backup portworx/px-backup --namespace <px-centralns> --create-namespace --set persistentStorage.enabled=true,persistentStorage.storageClassName=stork-snapshot-sc,operatorToChartUpgrade=true,pxbackup.orgName=<orgName>,pxcentralDBPassword=singapore,oidc.centralOIDC.defaultUsername=<px-central login>,oidc.centralOIDC.defaultPassword=<px-central-password>

File: content/Reference/install-helm-chart.md [hunk 12]
```diff
@@ -0,0 +1,92 @@
...
+
+The following table lists the configurable parameters of the PX-Backup Helm chart and their default values.
+
+Parameter | Description | Default
+--- | --- | ---
+`persistentStorage` | Persistent storage for all px-central components | `""`
+`persistentStorage.enabled` | Enable persistent storage | `false`
+`persistentStorage.storageClassName` | Provide storage class name which exists | `""`
+`storkRequired` | Scheduler name as stork | `false`
+`pxcentralDBUserName` | PX-Central cluster store mysql database username | `root`
```
Comment (tusharraut1994) [inline content/Reference/install-helm-chart.md:20]: We removed this field.
https://github.com/portworx/helm/tree/master/charts/px-backup#configuration

File: content/Reference/install-helm-chart.md [hunk 13]
```diff
@@ -0,0 +1,92 @@
...
+`oidc.externalOIDC` | Enable external OIDC provider | `""`
+`oidc.externalOIDC.enabled` | Enabled external OIDC provider | `false`
+`oidc.externalOIDC.clientID` | External OIDC client ID | default `""`
+`oidc.externalOIDC.clientSecret` | External OIDC client secret | `""`
+`oidc.externalOIDC.endpoint` | External OIDC endpoint | `""`
+`images` | PX-Backup deployment images | `""`
+`pxbackup` | Enable PX-Backup | `""`
+`pxbackup.enabled` | Enabled PX-Backup | `true`
+`pxbackup.orgName` | PX-Backup organization name | `default`
+`pxbackup.externalAccessHttpPort` | PX-Backup ui http port | `31234`
```
Comment (tusharraut1994) [inline content/Reference/install-helm-chart.md:41]: Remove this line.
https://github.com/portworx/helm/tree/master/charts/px-backup#configuration

File: content/Reference/install-helm-chart.md [hunk 14]
```diff
@@ -0,0 +1,92 @@
...
+`images.pxcentralBackendImage.imageName` | PX-Central backend image name | `pxcentral-onprem-ui-backend`
+`images.pxcentralBackendImage.tag` | PX-Central backend image tag | `1.1.1`
+`images.pxcentralMiddlewareImage.registry` | PX-Central middleware image registry | `docker.io`
+`images.pxcentralMiddlewareImage.repo` | PX-Central middleware image repo | `portworx`
+`images.pxcentralMiddlewareImage.imageName` | PX-Central middleware image name | `pxcentral-onprem-ui-lhbackend`
+`images.pxcentralMiddlewareImage.tag`| PX-Central middleware image tag | `1.1.1`
+`images.pxBackupImage.registry` | PX-Backup image registry | `docker.io`
+`images.pxBackupImage.repo` | PX-Backup image repo | `portworx`
+`images.pxBackupImage.imageName` | PX-Backup image name | `px-backup`
+`images.pxBackupImage.tag` | PX-Backup image tag | `1.0.2-rc1`
```
Comment (tusharraut1994) [inline content/Reference/install-helm-chart.md:64]: change the default value to 1.0.2

File: content/Reference/install-helm-chart.md [hunk 15]
```diff
@@ -0,0 +1,92 @@
...
+`images.pxcentralFrontendImage.imageName` | PX-Central frontend image name | `pxcentral-onprem-ui-frontend`
+`images.pxcentralFrontendImage.tag` | PX-Central frontend image tag | `1.1.1`
+`images.pxcentralBackendImage.registry` | PX-Central backend image registry | `docker.io`
+`images.pxcentralBackendImage.repo` | PX-Central backend image repo | `portworx`
+`images.pxcentralBackendImage.imageName` | PX-Central backend image name | `pxcentral-onprem-ui-backend`
+`images.pxcentralBackendImage.tag` | PX-Central backend image tag | `1.1.1`
+`images.pxcentralMiddlewareImage.registry` | PX-Central middleware image registry | `docker.io`
+`images.pxcentralMiddlewareImage.repo` | PX-Central middleware image repo | `portworx`
+`images.pxcentralMiddlewareImage.imageName` | PX-Central middleware image name | `pxcentral-onprem-ui-lhbackend`
+`images.pxcentralMiddlewareImage.tag`| PX-Central middleware image tag | `1.1.1`
```
Comment (tusharraut1994) [inline content/Reference/install-helm-chart.md:60]: change the default value to 1.1.2

File: content/Reference/install-helm-chart.md [hunk 16]
```diff
@@ -0,0 +1,92 @@
...
+`images.pxcentralApiServerImage.imageName` | API server image name | `pxcentral-onprem-api`
+`images.pxcentralApiServerImage.tag` | API server image tag | `1.0.4`
+`images.pxcentralFrontendImage.registry` | PX-Central frontend image registry | `docker.io`
+`images.pxcentralFrontendImage.repo` | PX-Central frontend image repo | `portworx`
+`images.pxcentralFrontendImage.imageName` | PX-Central frontend image name | `pxcentral-onprem-ui-frontend`
+`images.pxcentralFrontendImage.tag` | PX-Central frontend image tag | `1.1.1`
+`images.pxcentralBackendImage.registry` | PX-Central backend image registry | `docker.io`
+`images.pxcentralBackendImage.repo` | PX-Central backend image repo | `portworx`
+`images.pxcentralBackendImage.imageName` | PX-Central backend image name | `pxcentral-onprem-ui-backend`
+`images.pxcentralBackendImage.tag` | PX-Central backend image tag | `1.1.1`
```
Comment (tusharraut1994) [inline content/Reference/install-helm-chart.md:56]: change the default value to 1.1.2

File: content/Reference/install-helm-chart.md [hunk 17]
```diff
@@ -0,0 +1,92 @@
...
+`images.pullSecrets` | Image pull secrets | `docregistry-secret`
+`images.pullPolicy` | Image pull policy | `Always`
+`images.pxcentralApiServerImage.registry` | API server image registry | `docker.io`
+`images.pxcentralApiServerImage.repo` | API server image repo | `portworx`
+`images.pxcentralApiServerImage.imageName` | API server image name | `pxcentral-onprem-api`
+`images.pxcentralApiServerImage.tag` | API server image tag | `1.0.4`
+`images.pxcentralFrontendImage.registry` | PX-Central frontend image registry | `docker.io`
+`images.pxcentralFrontendImage.repo` | PX-Central frontend image repo | `portworx`
+`images.pxcentralFrontendImage.imageName` | PX-Central frontend image name | `pxcentral-onprem-ui-frontend`
+`images.pxcentralFrontendImage.tag` | PX-Central frontend image tag | `1.1.1`
```
Comment (tusharraut1994) [inline content/Reference/install-helm-chart.md:52]: change the default value to 1.1.2

File: content/Reference/install-helm-chart.md [hunk 18]
```diff
@@ -0,0 +1,92 @@
...
+`images.keycloakBackendImage.imageName` | PX-Backup keycloak backend image name | `postgresql`
+`images.keycloakBackendImage.tag` | PX-Backup keycloak backend image tag | `11.7.0-debian-10-r9`
+`images.keycloakFrontendImage.registry` | PX-Backup keycloak frontend image registry | `docker.io`
+`images.keycloakFrontendImage.repo` | PX-Backup keycloak frontend image repo | `jboss`
+`images.keycloakFrontendImage.imageName` | PX-Backup keycloak frontend image name | `keycloak`
+`images.keycloakFrontendImage.tag` | PX-Backup keycloak frontend image tag | `9.0.2`
+`images.keycloakLoginThemeImage.registry` | PX-Backup keycloak login theme image registry | `docker.io`
+`images.keycloakLoginThemeImage.repo` | PX-Backup keycloak login theme image repo | `portworx`
+`images.keycloakLoginThemeImage.imageName` | PX-Backup keycloak login theme image name | `keycloak-login-theme`
+`images.keycloakLoginThemeImage.tag` | PX-Backup keycloak login theme image tag | `1.0.1`
```
Comment (tusharraut1994) [inline content/Reference/install-helm-chart.md:84]: change the default value to 1.0.2

File: content/install.md (hunk 3 above)
Comment (bhavana-ar) [reply content/install.md]: No, We should remove this section

File: content/install.md [hunk 19]
```diff
@@ -11,55 +11,36 @@ series: backup
...
+    parameters:
+    repl: "3"
     ```
 
-## Install
+2. Generate the install spec through the **PX-Backup** [spec generator](https://central.portworx.com/specGen/wizard). 
 
-1. To install PX-Backup, generate the install spec through the **PX-Backup** [spec generator](https://central.portworx.com/specGen/wizard). 
-    <!-- if the step above is removed, remove this as well --> If you saved your cloud credentials as a Kubernetes secret ahead of time, enter the name and namespace of your secret.
+    If you're installing Portworx alongside PX-Backup, select the **Use storage class** checkbox under the **Storage** section of the **Spec Details** tab of the spec generator and enter the name of the storageclass you created in step 1 above.
```
Comment (bhavana-ar) [inline content/install.md:41]: "If you're installing Portworx alongside PX-Backup" is incorrect. Helm install is not installing PX. PX should already up up and running at this point.  It should be, i"f you are using portworx for px-backup installation", select the checkbox...

File: content/use-px-backup/upgrade.md (hunk 11 above)
Comment (nathan-px) [reply content/use-px-backup/upgrade.md:94]: @bhavana-ar So these all need to be changed by the user? what about `pxcentralDBPassword=singapore`?

Comment (bhavana-ar) [reply content/use-px-backup/upgrade.md]: Dont change ```pxcentralDBPassword=singapore``` and the rest (namespace, org name, default username and password) needs to be set by the user

File: content/use-px-backup/upgrade.md [hunk 20]
```diff
@@ -19,25 +23,13 @@ To upgrade PX-Backup from an operator-based install to the Helm-based install in
     curl -o create_new_pvc_from_snapshot.yaml https://github.com/portworx/helm/blob/chart_upgrade_specs/pxcentral-operator-to-chart-upgrade/create_new_pvc_from_snapshot.yaml
```
Comment (bhavana-ar) [inline content/use-px-backup/upgrade.md:23]: Curl command is incorrect: ```curl -o create_new_pvc_from_snapshot.yaml https://raw.githubusercontent.com/portworx/helm/chart_upgrade_specs/pxcentral-operator-to-chart-upgrade/create_new_pvc_from_snapshot.yaml```

File: content/use-px-backup/upgrade.md [hunk 21]
```diff
@@ -47,53 +39,68 @@ To upgrade PX-Backup from an operator-based install to the Helm-based install in
...
-4. Create new PVCs from the volume snapshots you created in step 1 in the same namespace where PX-Central on-prem was originally deployed:
+6. Create new PVCs from the volume snapshots you created in step 1 in the same namespace where PX-Central on-prem was originally deployed:
 
     ```console
-    $ kubectl apply -f create_new_pvc_from_snapshot.yaml
+    kubectl apply -f create_new_pvc_from_snapshot.yaml
     ```
 
-5. Install PX-Backup using the helm chart:
+7. Fetch the org name from your cluster, specifying the namespace in which your current on-prem PX-Central deployment exists:
```
Comment (bhavana-ar) [inline content/use-px-backup/upgrade.md:85]: This should move up. Either in the prerequisite section or as the first step.

**Pull request merged**